types/ibc.go

```diff
@@ -10,9 +10,60 @@ type IBCChannel struct {
 	CounterpartyEndpoint IBCEndpoint `json:"counterparty_endpoint"`
 	Order                IBCOrder    `json:"order"`
 	Version              string      `json:"version"`
-	// optional
-	CounterpartyVersion string `json:"counterparty_version,omitempty"`
-	ConnectionID        string `json:"connection_id"`
+	ConnectionID         string      `json:"connection_id"`
+}
+
+type IBCChannelOpenMsg struct {
+	OpenInit *IBCOpenInit `json:",omitempty"`
```

Comment:
These are currently (de)serialized into camel case names, since enum variants are camel case in _cosmwasm-std_. Should we change the representation to snake case for consistency in JSON fields?

Reply:
Good find! The new enums are missing an `#[serde(rename_all = "snake_case")]` for consistency. This should be changed in cosmwasm.